src/coupling/balance.hh

```diff
@@ -506,9 +493,9 @@ private:
     std::vector<std::vector<double> > fluxes_in_;
     std::vector<std::vector<double> > fluxes_out_;
     std::vector<std::vector<double> > masses_;
-    std::vector<std::vector<double> > sources_;
     std::vector<std::vector<double> > sources_in_;
     std::vector<std::vector<double> > sources_out_;
+    std::vector<double> temp_sources_;
```

Comment:
Is temp_sources_ used elsewhere than in calculate_cumulative()? If not, it should be a local variable in this method.

Reply:
done